<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication99Class</class>
 <widget class="QMainWindow" name="QtWidgetsApplication99Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication99</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>高铁订票系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="userInfoLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>558</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>车票查询</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QLineEdit" name="toStationEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="fromStation">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>出发站：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="toStation">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>到达站：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="startdate">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>出发日期：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>查询车票</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="fromStationEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QDateEdit" name="dateEdit">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="orderGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>276</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>我的订单</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="orderTable"/>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>取消订单</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bookButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>预定车票</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="adminToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStationManage"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStationManage">
   <property name="text">
    <string>车站管理</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsApplication99.qrc"/>
 </resources>
 <connections/>
</ui>
